src/components/dashboard/main-content/CourseRecommendations.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react';
+import { Button } from '@edx/paragon';
+import { Link } from 'react-router-dom';
+import { AppContext } from '@edx/frontend-platform/react';
+
+import SkillsQuizImage from '../../../assets/images/skills-quiz/skills-quiz.png';
+
+const CourseRecommendations = () => {
```

Comment:
Any plan to write tests for this component?

Reply:
Done